packages/react/src/components/Link/Link-test.js

```diff
@@ -56,4 +63,55 @@ describe('Link', () => {
       expect(link.props().rel).toEqual('noopener');
     });
   });
+
+  describe('passes automated automated verification testing', () => {
+    it('should have no Axe violations', async () => {
+      render(
+        <Link href="/" className="some-class">
+          A simple link
+        </Link>
+      );
+
+      await expect(screen.getByText('A simple link')).toHaveNoAxeViolations();
+    });
+
+    it('should have no Accessibility Checker violations', async () => {
+      render(
+        <main>
+          <Link href="/" className="some-class">
+            A simple link
+          </Link>
+        </main>
+      );
+
+      await expect(screen.getByText('A simple link')).toHaveNoACViolations(
+        'Link'
+      );
+    });
+  });
+
+  describe('should be keyboard accessible', () => {
```

Comment:
Maybe this:

```suggestion
  describe('keyboard support', () => {
```

?

Another option could be to put it at the top-level without needing to put it in a keyboard block (e.g. it's just part of the component behavior)

For context, the `should be ...` convention typically is paired with the `it` block to describe the "thing" that is being tested.